streamlitDemo/utility.py

```diff
@@ -290,7 +294,7 @@ def autoplay_video(video_file_path):
 
 def autoplay_live_video(video_url):
     md = f"""
-        <iframe width="550" height="400" src="{video_url}?autoplay=1">
+        <iframe width="550" height="400" src="{video_url}?autoplay=1&mute=1">
```

Comment:
this change is for allow for autoplay. Without the mute parameter the autoplay will not work per Google's guidelines https://developer.chrome.com/blog/autoplay/#new-behaviors